blocks/adp-share-header/adp-share-header.js

```diff
@@ -0,0 +1,43 @@
+
+import createConfirmDialog from '../../scripts/confirm-dialog.js';
```

Comment:
Is this import needed?

Reply:
Removed scripts/confirm-dialog.js it is not used, probably a copied over from the adp-collection-header.js